airbyte-config/models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -113,6 +115,17 @@ public String getDatabaseUrl() {
     return getEnsureEnv(DATABASE_URL);
   }
 
+  @Override
+  public String getWebappUrl() {
+    final String webappUrl = getEnv.apply(WEBAPP_URL);
+    if (!Strings.isEmpty(webappUrl)) {
+      return webappUrl;
+    }
+
+    LOGGER.info(WEBAPP_URL + " not found, defaulting to: http://localhost:8000/");
+    return "http://localhost:8000/";
```

Comment:
I am not sure if the default should be there or in the `.env`.

Reply:
I hesitated with putting the default in the docker-compose when defining the env at first

But there are other default values being fallback to in this file:
- WORKSPACE_DOCKER_MOUNT
- LOCAL_DOCKER_MOUNT
- DOCKER_NETWORK
- TRACKING_STRATEGY
- WORKER_ENVIRONMENT

That's why I chose to put it here too in the end...

What's preferable then?